plugins/parsers/registry.go

```diff
@@ -66,107 +66,107 @@ type Parser interface {
 // and can be used to instantiate _any_ of the parsers.
 type Config struct {
 	// Dataformat can be one of: json, influx, graphite, value, nagios
-	DataFormat string `toml:"data_format"`
+	DataFormat string `toml:"data_format" json:"data_format"`
```

Comment:
Duplicating the toml name for json is extra work and error prone. Could we use something like the retag module (https://github.com/sevlyar/retag) to copy at runtime?

Are plugins also going to need to do this?

Reply:
it does add some clarity to what is going on. Plugins don't need to do this because we check the toml tags. We could potentially do that here, but I think there was a reason for it. maybe the convenience of being able to use the default unmarshaller.